bgpd/bgp_route.c

```diff
@@ -4310,6 +4313,7 @@ static void eoiu_marker_queue_free(struct meta_queue *mq, struct bgp_dest_queue
 		XFREE(MTYPE_BGP_EOIU_MARKER_INFO, dest->info);
 		STAILQ_REMOVE_HEAD(l, pq);
 		STAILQ_NEXT(dest, pq) = NULL; /* complete unlink */
+		XFREE(MTYPE_BGP_NODE, dest);
```

Comment:
is there a similar problem in `other_meta_queue_free()` - that's not un-ref'ing or freeing the dests either

Reply:
there should not be.  The eoiu marker is the only one where a `fake` dest is created that needs to be deleted on the other side.